### DISCUSSION TITLE
Append Speaker to existing Group

### DISCUSSION THREAD

**Discussion by blackmore190** [B5r1oJ0A9G/teufel_raumfeld#24] [Q&A]
Hi, 

how is it possible to append a speaker to an existing group?
I want to add "TV" to the already playing group of "Living Room" an "Kitchen"

If i use this: 
```
service: teufel_raumfeld.group
data:
  room_names:
    - Kitchen
    - Living room
    - TV
```

I get a "new" group with no media loaded.

I hope you understand what I am looking for, 

Thanks for your help!

**Comment by B5r1oJ0A9G** [comment]
Hi @blackmore190,

This is not yet implemented in ``teufel_raumfeld``. Hoever, I implemented it in ``hassfeld`` so should be no big deal to implement. Feel free to create a feature request in the Issue tracke and I'll follow-up on it as soon as I have some spare time. Right now I'm quite busy but this would be a welcomed opportunity to get a break ;)

Would you be willing to contribute to the documentation of the new feature in the [Wiki](https://github.com/B5r1oJ0A9G/teufel_raumfeld/wiki) once implemented?

**Reply by blackmore190** [reply]
Hi,
Yes I will contribute to the documentation. If possible i could also contribute in the Implementation. My Python skills are pretty Basic, nur i will give my very best ;)

**Comment by B5r1oJ0A9G** [comment]
Implemented by  #25.
